### ISSUE TITLE
Can the input be VCF file from other SV callers?

### ISSUE THREAD

**Issue by clifflun** [120L021101/MEHunter#1]
Can the input be VCF file from other SV callers?

**Comment by 120L021101** [comment]
No, because it needs supporting reads in vcf "info", which, within ny knowledge, only cuteSV has, and it needs several sharp symbols to split the read names, if u find other sv caller has similar info, u can manually add the sharp symbols and it hould be runnable
